vault/auth.go

```diff
@@ -457,6 +461,9 @@ func (c *Core) teardownCredentials() error {
 // newCredentialBackend is used to create and configure a new credential backend by name
 func (c *Core) newCredentialBackend(
 	t string, sysView logical.SystemView, view logical.Storage, conf map[string]string) (logical.Backend, error) {
+	if alias, ok := credentialAliases[t]; ok {
```

Comment:
@joelthompson That's a perfectly fine way to go and that is what I meant. Sorry that I wasn't clear enough. Role tags V2 is something that we can tackle later.